docs/guidance/text-summarizer-rayservice.md

```diff
@@ -0,0 +1,71 @@
+# Serve a text summarizer using RayService
+
+> **Note:** The Python files for the Ray Serve application and its client are in the [ray-project/serve_config_examples](https://github.com/ray-project/serve_config_examples) repo.
+
+## Step 1: Create a Kubernetes cluster with GPUs
+
+Follow [aws-eks-gpu-cluster.md](./aws-eks-gpu-cluster.md) or [gcp-gke-gpu-cluster.md](./gcp-gke-gpu-cluster.md) to create a Kubernetes cluster with 1 CPU node and 1 GPU node.
+
+## Step 2: Install KubeRay operator
+
+Follow [this document](../../helm-chart/kuberay-operator/README.md) to install the latest stable KubeRay operator via Helm repository.
+Please note that the YAML file in this example uses `serveConfigV2`, which is supported starting from KubeRay v0.6.0.
+
+## Step 3: Install a RayService
+
+```sh
+# path: ray-operator/config/samples/
+kubectl apply -f ray-service.text-sumarizer.yaml
+```
+
+This RayService configuration contains some important settings:
+
+* Its `tolerations` for workers match the taints on the GPU node group. Without the tolerations, worker Pods won't be scheduled on GPU nodes.
```

Comment:
I've realized the statement was inaccurate. A Pod with tolerations can be scheduled on nodes without any taints or on nodes with specific taints. Hence, the primary objective of adding a taint to the GPU node is to prevent Pods (e.g. head Pod & KubeRay operator Pod) without the appropriate tolerations from being scheduled on the GPU nodes, rather than ensuring that Pods with the toleration are scheduled on the GPU nodes. We also need to update the statement in stable diffusion doc.